.devcontainer/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM mcr.microsoft.com/devcontainers/dotnet:0-6.0-bullseye
```

Comment:
I couldn't find an explanation of the tag. Shouldn't this target a major or minor or is `0-6.0-bullseye` already a sliding tag?

It's not entirely sure to me. I looked at the following pages:

- https://hub.docker.com/_/microsoft-vscode-devcontainers
- https://mcr.microsoft.com/v2/vscode/devcontainers/dotnet/tags/list

Reply:
Yeah this is all a bit magical. I think I could also remove the bullseye again because I had it in place for some trials but don´t need it anymore I think

Reply:
Switch to generic tag and added links